hacks/iptables/buildvm-scripts/canttouchthat.py

```diff
@@ -216,6 +223,11 @@ def main(output_networks, enforce, dry_run, clean):
         print('Would have written the following direct rules to {}'.format(DIRECT_XML_PATH))
         print_direct_rules(direct)
 
+def get_fastly_ip_ranges():
+    response = requests.get("https://api.fastly.com/public-ip-list")
```

Comment:
are we guaranteed that the firewall allows `api.fastly.com`?

Also, probably wise to allow some retries.

But we can watch and see if any refinement is needed.